homeassistant/components/shelly/coordinator.py

```diff
@@ -543,7 +543,7 @@ def _async_handle_update(self, device_: RpcDevice, update_type: UpdateType) -> N
         elif update_type is UpdateType.DISCONNECTED:
             self.hass.async_create_task(self._async_disconnected())
         elif update_type is UpdateType.STATUS:
-            self.async_set_updated_data(self.device)
+            self.async_set_updated_data(None)
```

Comment:
`coordinator.data` is never used actually. The device is accessed via `coordinator.device` instead.

Leaving the call in place to properly schedule the next update.